<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1644</width>
    <height>1488</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>이순신 돋움체 M</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AI Inventor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../image/logo.png</normaloff>../image/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(241, 242, 246);</string>
  </property>
  <widget class="QPushButton" name="connectBtn">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>15</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 15px;

</string>
   </property>
   <property name="text">
    <string>연결</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>470</y>
     <width>361</width>
     <height>981</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent</string>
   </property>
   <widget class="QPushButton" name="translationBtn">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>20</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>125</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>번역</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ttsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>텍스트➜음성</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sttBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>음성➜텍스트</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ocrBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>이미지
텍스트 인식</string>
    </property>
   </widget>
   <widget class="QPushButton" name="objBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>  이미지  
물체 인식</string>
    </property>
   </widget>
   <widget class="QPushButton" name="faceBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>820</y>
      <width>341</width>
      <height>150</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{font: 15pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 30px;
padding-right:30px;
text-align: right;}
QPushButton:hover{
            font: 15pt &quot;이순신 돋움체 M&quot;;
            border-radius: 30px;
            background-color:  rgb(52, 73, 94);
            color: rgb(255, 255, 255);
            }</string>
    </property>
    <property name="text">
     <string>  유명인
얼굴 인식</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>44</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/Translation.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/tts.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/stt.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/ocr.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/obj.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>845</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/celebrity.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="output">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>1070</y>
     <width>1211</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 20pt &quot;이순신 돋움체 M&quot;;
border-radius: 20px;
background-color: rgb(250, 250, 250);</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'이순신 돋움체 M'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt; &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>1360</x>
     <y>15</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 15px;

</string>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBtn">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>15</y>
     <width>125</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: rgb(223, 228, 234);
	border-radius: 15px;


</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inputBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>1211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 20pt &quot;이순신 돋움체 M&quot;;
            border-radius: 20px;
            background-color:  rgb(223, 228, 234);
            color:rgb(52, 73, 94);
}
QPushButton:hover{font: 20pt &quot;이순신 돋움체 M&quot;;
	color: rgb(52, 73, 94);
	background-color: white;
	border-radius: 20px;
}</string>
   </property>
   <property name="text">
    <string>입력</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>1211</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="aiSelect"/>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>20</y>
     <width>391</width>
     <height>1451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(206, 214, 224);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>270</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>420</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 20pt &quot;이순신 돋움체 M&quot;;
color: rgb(52, 73, 94);</string>
   </property>
   <property name="text">
    <string>AI 모델 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>721</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 12pt &quot;이순신 돋움체 m&quot;;
color: rgb(52, 73, 94);</string>
   </property>
   <property name="text">
    <string>이용 방법: ① 입력 ➜ ② AI 모델 선택 ➜ ③ 연결 or 저장 or Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageview">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>501</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>1211</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width: 5px;
border-style: dashed;
border-radius:10px;

border-color:rgb(206, 214, 224);


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textview">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>1101</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;이순신 돋움체 M&quot;;
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fileview">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>240</y>
     <width>611</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;이순신 돋움체 M&quot;;
background:transparent;
color: rgb(52, 73, 94);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>frame</zorder>
  <zorder>connectBtn</zorder>
  <zorder>output</zorder>
  <zorder>saveBtn</zorder>
  <zorder>clearBtn</zorder>
  <zorder>inputBtn</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>textview</zorder>
  <zorder>fileview</zorder>
  <zorder>imageview</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
